telegram/ext/_updater.py

```diff
@@ -345,7 +344,7 @@ async def polling_action_cb() -> bool:
 
             if updates:
                 if not self.running:
-                    self._logger.critical(
+                    _logger.critical(
                         "Updater stopped unexpectedly. Pulled updates will be ignored and again "
```

Comment:
This is unrelated to the changes but something seems to be wrong with the phrase on the next line ("Pulled updates will be ignored and again..."). Maybe a verb is missing?